neuro_extras/main.py

```diff
@@ -330,11 +336,12 @@ async def _data_cp(
             container_src_uri = source_url
 
         if destination_url_type == UrlType.STORAGE:
-            volume.append(f"{str(destination_url)}:/var/storage")
-            container_dst_uri = URL("/var/storage")
+            volume.append(f"{str(destination_url.parent)}:/var/storage")
+            container_dst_uri = URL(f"/var/storage/{destination_url.name}")
         elif destination_url_type == UrlType.DISK:
-            volume.append(f"{str(destination_url)}:/var/disk:rw")
-            container_dst_uri = URL("/var/disk/")
+            disk_ref = str(destination_url)[:46]  # disk:$ID (which is 41 symbols long)
```

Comment:
wow this one is hacky.. Why are we sure that `destination_url` is in the format of `disk:$ID`? At least this should have been extracted into a separate method with the format check (fail if it's in a diff format). Let's cover it in a following-up PR.

Reply:
also: `destination_url` is of type `yarl.URL`, so we should have manipulated `destination_url.path`  instead of `str(destination_url)`

Reply:
does `neuro_api` have utilities for parsing disk URIs ?

Reply:
>also: destination_url is of type yarl.URL, so we should have manipulated destination_url.path instead of str(destination_url)

Actually, it does not changes the code dramatically since, for volumes, we need a ref with the scheme (disk:$ID, not only $ID). If I would use `path`, it would be smth like:
```python
            disk_id = str(destination_url.path)[:41]  # disk:$ID (which is 41 symbols long)
            volume.append(f"disk:{disk_id}:/var/disk:rw")
            container_dst_uri = URL(f"/var/disk/{str(destination_url.path)[42:]}")
```
wdyt?

> does neuro_api have utilities for parsing disk URIs ?

They do, but AFAIR, @anayden decided to write our own parser, since neuro_api's one doesn't fit to our needs. Cannot elaborate deeper, but pbbly @anayden could..

Reply:
added suggestion above in 54d8a15